IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_RailtypeGuardRail/DocPropertySet.xml

```diff
@@ -1,11 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RailtypeGuardRail" UniqueId="f588317f-9341-b716-2999-2c57a31d92a2" ApplicableType="IfcRail/GUARDRAIL" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RailTypeGuardRail" UniqueId="f588317f-9341-b716-2999-2c57a31d92a2" ApplicableType="IfcRail/GUARDRAIL" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
FILE NAME Issue, please adjust the casing in the file name so it is correct

Reply:
Thanks for pointing it out, but I don't have this issue anymore locally. Github is not case-sensitive for file names..

Reply:
wierd, will be ok for the merge will sort after